test/libyul/yulOptimizerTests/fakeStackLimitEvader/connected.yul

```diff
@@ -29,31 +29,32 @@
 //     {
 //         a := 21
 //         mstore(0x60, 1)
-//         let b_1, a_2, $c_3 := z()
-//         mstore(0x60, $c_3)
-//         a := a_2
-//         b := b_1
+//         let b_8, a_9, $c_10 := z()
+//         mstore(0x60, $c_10)
+//         a := a_9
+//         b := b_8
 //     }
 //     function f() -> x
 //     {
-//         mstore(0x20, 0)
+//         let $x2_11
+//         mstore(0x20, $x2_11)
```

Comment:
Actually to be precise: this is an actual side-effect of the change other than variable indices - but it's expected and fine (it's due to the removal of the special handling of single-variable declarations - if there was an initial value, the expression joiner would inline it, but since there is none, it stays like this, but there's no harm in it)